Makefile

```diff
@@ -177,6 +177,9 @@ crds: $(CONTROLLER_GEN) $(YQ)
 	@echo Updating CRD manifests
 	@build/crds/build-crds.sh $(CONTROLLER_GEN) $(YQ)
 
+helm-gen-rbac: ## generate RBAC from Helm charts
+	bash build/rbac/get-helm-rbac.sh
+
```

Comment:
With `make helm-gen-rbac`, `make codegen`, and `make crds` and maybe some other commands, I wonder if we could use a `make precheckin` or similar command so all these extra commands can be run from a single place before pushing and finding out the CI fails. It's really a separate question from this PR though.

Reply:
I think that would be a good idea, also separate.

Reply:
I'll do this once `csv-template` and `gen-rbac` both exist.